### ISSUE TITLE
Edge length and width from attribute

### ISSUE THREAD

**Issue by TechnicallyButter** [WestHealth/pyvis#143]
I was wondering if it was possible to set edge length from attribute in pyvis while also having that length show up on the edge ?
The graph I'm trying to visualize is from networkx (e.g: 'node1', 'node2', 'len', 'width') and is around 2000 nodes.

**Comment by TechnicallyButter** [comment]
No, not edge labels but rather the distance between nodes. I want to set the actual distance between any two nodes to be the 'len' attributes of the graph.

**Comment by jhunpingco** [comment]
I think you would manually have to the positions of the nodes that reflect your desired inter-node distance. There's nothing in PyVis to automatically do this.

**Comment by TechnicallyButter** [comment]
That's a bummer :(
Thanks for you answer !
